src/test/java/seedu/address/logic/parser/ArgumentTokenizerTest.java

```diff
@@ -15,12 +21,43 @@
     private final Prefix hatQ = new Prefix("^Q");
 
     @Test
-    public void tokenize_emptyArgsString_noValues() {
+    public void tokenize_emptyArgsString_noValues() throws ParseException {
         String argsString = "  ";
```

Comment:
As mentioned above, I think `throw ParseException` should be removed if `ParseException` in `ArgumentTokenizer#findAllPrefixPositions()` is never thrown.

Reply:
It can be thrown. I will look into this.